src/lib/select/select.md

```diff
@@ -92,3 +92,11 @@ Here are the available global options:
 - <kbd>DOWN_ARROW</kbd>: Focus next option
 - <kbd>UP_ARROW</kbd>: Focus previous option
 - <kbd>ENTER</kbd> or <kbd>SPACE</kbd>: Select focused item
+
+### Accessibility
+The select component without text or label should be given a meaningful label via
```

Comment:
FYI, the placeholder, hint, and error stuff will work the same way as input after the refactor. For now this is fine though